Test/BaseTest.cs

```diff
@@ -268,6 +268,7 @@ public Subscription CreateNewRampSubscription(int numberOfRamps)
         public Plan CreateNewRampPlan(int numberOfRamps)
         {
             var plan = new Plan(GetMockPlanCode(), GetMockPlanName()) { Description = "Create Ramp Plan" };
+            plan.TaxExempt = true;
```

Comment:
What's the reason for this change?

Reply:
Because in the prod site, if i dont do this then taxes are included in the Totals coming back and therefore they dont match the first ramp value

Reply:
Comes from this in the AC:

```
Associated plans have NO tax.
Associated purchases have NO gift cards or other discounts
```